travel_journey/tests/test_journey.py

```diff
@@ -194,3 +197,12 @@ def test_inv_estimate_date(self):
             cr, uid, journey_id,
             field_name='date_stop', val='2014-03-10',
             arg=None, context=context)
+
+    def test_get_passenger(self):
+        """ Test the  _get_passenger function defined in the report """
+        cr, uid, vals, context = self.cr, self.uid, self.vals, self.context
+        journey_id = self.journey_model.create(cr, uid, vals, context=context)
+        journey_rec = self.journey_model.browse(
+            self.cr, self.uid, journey_id, context=self.context)
```

Comment:
@ehdem Since you declared you local copies of cr, uid and context its better to use them instead of the class ones.